documentation/design-docs/dotnet-tools.md

```diff
@@ -1,124 +1,797 @@
+
+    Multi-purpose tool did have a few advantages we are giving up:
+    (a) Size on disk would have been better. Each tool must include the closure of its non-framework dependencies which means if any tools share a dependency (symbol loading for example) there will be multiple copies of that assembly in the global tools install directory
+    (b) A single dotnet install command. We could improve this in dotnet tool install itself, for example by allowing multiple tools to be listed in an install command (similar to many package managers) or supporting a tool meta-package that is just a grouping of related tools. In the meantime we should try to avoid needing too many different tools in the same workflow.
+
+3. Do we have a stripped down stand-alone collector?
+
+    Not at this time, though I still advocate for good layering. There are at least two useful scenarios for this I could see, one is a persistent monitoring agent and the other is on-demand install by support engineers in response to a production problem. In the latter case size on disk is probably only a small concern relative to being able to isolate and customer confidence in the tools. If we could lose the SDK dependency we'd probably be in decent shape there. In the former case size does matter more, but we again be better optimizing for no SDK dependency and FDD deployment before eliminating the analysis components becomes the top size issue. There are also other non-size concerns if we ultimately go this path such as daemonizing the tool, log cycling, auto-triggered collections and remote administration.
+
+4. Do we support command line response files?
+
+    Not at this time. I didn't see any documentation suggesting that dotnet supports the response file generally and it would be nice to follow suit rather than rolling our own. We can revisit this based on customer feedback.
+
+5. Do we support '/' style args that are more common on windows or only '--' style args?
+
+    Not at this time (feel like a broken record yet?). We should document and parse all arguments accepting only the single dash or double dash form. For example -h or --help are recognized for help, but /help is not. This keeps us identical to the behavior of other dotnet tools.
+    FWIW there might be value in accepting the /arg form of arguments but I'd rather it gets taken up across all dotnet tools or as a generic feature of the command-line parsing library so that we have some degree of standardization. Even if we did start recognizing the /arg form, I still suggest only printing the -/-- forms in the help to prevent clutter.
+
```

Comment:
This question has been discussed as well for `System.CommandLine`: https://github.com/dotnet/command-line-api/issues/351. (While we currently can support both forms, they have to be configured verbatim until or unless this feature is taken up.) 

The idea of limiting what's shown in help but still honoring [other forms](https://github.com/dotnet/command-line-api/blob/c9450cba187e377f8c58c5a60d07767c5ebe6b9c/src/System.CommandLine/Invocation/InvocationExtensions.cs#L262) makes sense because it helps users who don't know the tool find help as easily as possible.